pkg/test/scenario/example_noop_test.go

```diff
@@ -22,10 +23,9 @@ var noopScenario Scenario = Scenario{
 		},
 	},
 	Spec: model.Spec{
-		Engine: model.EngineNoop,
-		Wasm: model.JobSpecWasm{
-			EntryPoint: "_start",
-		},
+		// TODO(forrest): [correctness] this doesn't make sense, why wasm?
+		EngineDeprecated: model.EngineNoop,
+		EngineSpec:       model.NewWasmEngineSpec(model.StorageSpec{}, "_start", nil, nil, nil),
```

Comment:
Not sure what the correct change is here, I'd assume replacing wasm with noop would be fine?